makefile

```diff
@@ -70,6 +70,7 @@ build:
 	@ echo "\n${CHECK} Done"
 
 	@ echo "${HR}\nCopy Proxima Nova (if available)...${HR}"
+	@ mkdir ./fonts
```

Comment:
Fixes a make issue where an error was thrown if the `/fonts` directory didn't exist, I thought `touch` would create the directory but it doesn't...